extlinks/common/helpers.py

```diff
@@ -241,6 +241,13 @@ def filter_queryset(queryset, filter_dict):
                 on_user_list=True
             )
 
+    if 'namespace_id' in filter_dict:
+        namespace_id = filter_dict['namespace_id']
```

Comment:
Could you please name this variable something else? for example ns_id

Reply:
Hi @Jain-Aditya, I can but I personally think `namespace_id` is more descriptive and useful than `ns_id`.

Reply:
The idea is basically not to use the shorter name but a name other than `namespace_id` because `namepace_id` is already holding a dictionary key and it might appear a bit confusing. Also we are not using it anywhere else so keeping a shorter name would be no harm. :)

Reply:
Dictionary keys are used in variable names multiple times in the same method in the exact same way.

Reply:
I agree @supertassu. Its just a paradigm that I follow. You are right. Apologies :)